zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -830,4 +839,9 @@ public static Paragraph fromJson(String json) {
     return Note.getGson().fromJson(json, Paragraph.class);
   }
 
+  @Override
+  public void postJsonDeser() {
```

Comment:
When this method is called ?

Reply:
Missed commiting of zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java. Have a look there.

Reply:
we can call `parseText` in  method `fromJson` which is much easier. `fromJson` is the method which is called when derealization.

Reply:
adjusted code according to your comment. Can you review again please.